sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -994,14 +989,8 @@ def download_blob(self, repository: str, digest: str, **kwargs) -> DownloadBlobR
         :param str digest: The digest of the blob to download.
         :returns: DownloadBlobResult
         :rtype: ~azure.containerregistry.DownloadBlobResult
-        :raises ValueError: If the parameter repository or digest is None.
         """
-        try:
-            deserialized = self._client.container_registry_blob.get_blob(repository, digest, **kwargs)
-        except Exception:
-            if repository is None or digest is None:
```

Comment:
Same as `download_manifest()`, will get `ValueError: No value for given attribute` when repository or digest is None.